server/src/main/java/com/thoughtworks/go/server/service/PluginProfilesService.java

```diff
@@ -88,7 +86,9 @@ private void addErrorsToConfiguration(ValidationResult result, PluginProfile new
 
     protected void update(Username currentUser, M pluginProfile, LocalizedOperationResult result, PluginProfileCommand command) {
         try {
-            validatePluginProperties(command, pluginProfile);
+            if (command.shouldValidatePluginProperties()) {
```

Comment:
I am wondering if the Command should decide bout the plugin validation. This is more a behaviour of the PluginProfileService, how bout `update` taking a flag `validatePluginProperties` and consider validation based on it.